stylesheet/styles.css

```diff
@@ -18,38 +18,99 @@ h1 {
 }
 
 .choices {
-    display: flex;
-    justify-content: center;
-    gap: 100px;
+  display: flex;
+  justify-content: center;
+  gap: 100px;
 }
 
 button {
```

Comment:
Applying styling directly to a tag is generally not great practice. This should use a class instead.

Reply:
Is this more of a good practice suggestion? In the event there's only one tag or a tag that requires the same styling in different areas would it be acceptable? Does it come down to maintainability?

Reply:
Its about maintainability and controlling side affects. If we ever wanted to add another button with different styling then we would have to override this styling which is a bit hacky